scripts/buildorder.py

```diff
@@ -101,13 +116,16 @@ def recursive_dependencies(self, pkgs_map):
 
 class TermuxSubPackage:
     "A sub-package represented by a ${PACKAGE_NAME}.subpackage.sh file."
-    def __init__(self, subpackage_file_path, parent):
+    def __init__(self, subpackage_file_path, parent, fast_build_mode, virtual=False):
```

Comment:
@tomty89 Could you document/explain the virtual boolean somewhere? How should it be used?

Reply:
Hmm it isn't really something new from me.

In case you missed it, it's for the condition here:
https://github.com/termux/termux-packages/blob/28711ba75103d12301e4852e8605ac8fa227a465/scripts/buildorder.py#L125-L126

Which merely "implements" this in buildorder.py:
https://github.com/termux/termux-packages/blob/28711ba75103d12301e4852e8605ac8fa227a465/scripts/build/termux_create_subpackages.sh#L8-L12

It's named virtual because of:
https://github.com/termux/termux-packages/blob/28711ba75103d12301e4852e8605ac8fa227a465/scripts/build/termux_create_subpackages.sh#L4

Reply:
Also see:
https://github.com/termux/termux-packages/blob/28711ba75103d12301e4852e8605ac8fa227a465/scripts/buildorder.py#L92-L95